src/System.Net.Sockets.Legacy/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -2140,7 +2222,7 @@ public void BeginReceiveMessageFromV6BoundToAnyV4_NotReceived()
         }
 
         [Fact]
-        [ActiveIssue(4004, PlatformID.AnyUnix)]
+        [ActiveIssue(4004, PlatformID.OSX)]
```

Comment:
Nothing changed in the implementation to enable this, right?  The ActiveIssue was just too broad initially?